CodeGeneration/Sources/generate-swiftparser/templates/DeclarationModifierFile.swift

```diff
@@ -24,20 +24,23 @@ let declarationModifierFile = SourceFile {
     """
   )
   
-  EnumDecl("enum DeclarationModifier: SyntaxText, ContextualKeywords, RawTokenKindSubset") {
+  EnumDecl("enum DeclarationModifier: RawTokenKindSubset") {
     for attribute in DECL_MODIFIER_KINDS {
-      EnumCaseDecl("case \(raw: attribute.swiftName) = \"\(raw: attribute.name)\"")
+      EnumCaseDecl("case \(raw: attribute.swiftName)")
     }
     InitializerDecl("init?(lexeme: Lexer.Lexeme)") {
-      SwitchStmt(switchKeyword: .switch, expression: Expr("lexeme.tokenKind")) {
-        for attribute in DECL_MODIFIER_KINDS where attribute.swiftName.hasSuffix("Keyword") {
-          SwitchCase("case .\(raw: attribute.swiftName):") {
-            SequenceExpr("self = .\(raw: attribute.swiftName)")
+      SwitchStmt(switchKeyword: .switch, expression: Expr("lexeme")) {
```

Comment:
Think we could remove the `.switch` keyword

Reply:
Thanks